### DISCUSSION TITLE
Error syncing animation in HERESY mod. Invisible female leather chestguard.

### DISCUSSION THREAD

**Discussion by GVOZDJ** [flareteam/flare-game#950] [Q&A] [answered]
I have downloaded the game packages using synaptic. Then i've coppied HERESY folder to games mods directory. I't played well until i bought a leather chestguard from blacksmith. The heroes body became invisible and the terminal repoted an error:

**ERROR: Avatar: Error syncing animation in 'animations/avatar/female/Leather_chestguard.txt' to 'animations/hero.txt'.**

Also it reports other error messages all the time, such as:

_ERROR: [/usr/share/games/flare/mods/HERESY/animations/npcs/MalePeasant1.txt:24] AnimationSet: Frame index (16) is out of bounds [0, 16].
ERROR: [/usr/share/games/flare/mods/HERESY/npcs/PigSE.txt:3] StatBlock: Warning! 'gfx' is deprecated. Use 'animations' instead.
ERROR: [/usr/share/games/flare/mods/HERESY/npcs/PigSW.txt:3] StatBlock: Warning! 'gfx' is deprecated. Use 'animations' instead.
ERROR: [/usr/share/games/flare/mods/HERESY/npcs/Lux_priest.txt:4] StatBlock: Warning! 'gfx' is deprecated. Use 'animations' instead.
ERROR: [/usr/share/games/flare/mods/HERESY/animations/npcs/LuxPriestNpc.txt:18] AnimationSet: Frame index (7) is out of bounds [0, 7]._
![image](https://github.com/flareteam/flare-game/assets/118244982/8afeeb90-b0ec-4fd9-854c-8a9ba34d9500)

Any ideas what could be the reason and how to resolve it?

**Comment by dorkster** [comment, marked as answer]
Looks like a case-sensitivity issue. The easiest fix for this specific issue would be to copy:

    animations/avatar/female/leather_chestguard.txt

to

    animations/avatar/female/Leather_chestguard.txt

(note the capital 'L')

Modders on Windows don't notice when this happens, so it's not the first time this has cropped up.

**Reply by GVOZDJ** [reply]
Oh god. It's so dumb :D It actually worked. For some reason files names in male and female folders are different.
